sceptre/cli/dump.py

```diff
@@ -53,3 +55,55 @@ def dump_config(ctx, path):
     else:
         for config in output:
             write(config, output_format)
+
+
+@dump_group.command(name="template")
+@click.argument("path")
+@click.option(
+    "-n",
+    "--no-placeholders",
+    is_flag=True,
+    help="If True, no placeholder values will be supplied for resolvers that cannot be resolved.",
+)
+@click.pass_context
+@catch_exceptions
+def dump_template(ctx, no_placeholders, path):
```

Comment:
How is this different from `generate`?

Reply:
@jfalkenstein This was what @zaro0508 and I discussed in the long thread you may not have read! The problem with `generate` always has been that it had no way to associate templates with stacks in its output for multiple stacks in a stack group, but rather just randomly orders a bunch of yaml docs that the caller has to figure out. In order to implement #1318 however we need to know which stack goes with which doc. So we agreed to deprecate `generate` and create `dump template` which behaves similarly to `dump config`.

Reply:
Hmmm... I'm not crazy about the idea of deprecating `generate`. Can't we just fix it to make it better?

Reply:
Yeah, I use the heck out of generate.

Reply:
Be sure to consider all this in the context of how the new naming allows such a sane and easy to understand structure for the output templates to emerge https://github.com/alexharv074/sceptre/pull/3 

I think it is unavoidable that if we keep the existing naming (`dump config` which was named by Jon and I after a _lot_ of discussion(!) vis-a-vis `generate`) the output templates naming is going to be a dog's breakfast.